scripts/globals/mobskills.lua

```diff
@@ -219,9 +223,15 @@ xi.mobskills.mobPhysicalMove = function(mob, target, skill, numberofhits, accmod
 
     -- Set everything to 1 because the FTP for mobs iis not supposed to be for attack only.
     -- TODO: Apply attack modifiers to certain mobskills with params
-    local params = { atk000 = 1, atk150 = 1, atk300 = 1 }
+    local params       = { atk000 = 1, atk150 = 1, atk300 = 1 }
+    local paramsRanged = { atk100 = 1, atk200 = 1, atk300 = 1 }
     -- Getting PDIF
-    local pdifTable = xi.weaponskills.cMeleeRatio(mob, target, params, 0, mob:getTP(), xi.slot.main)
+    local pdifTable = xi.weaponskills.cMeleeRatio(mob, target, params, 0, mob:getTP(), xi.slot.MAIN)
+
+    if tpeffect == xi.mobskills.magicalTpBonus.RANGED then
+        pdifTable = xi.weaponskills.cRangedRatio(mob, target, paramsRanged, 0, mob:getTP())
+    end
+
```

Comment:
The funny thing is...mobs (and players) dont even have attack modifiers for ranged attacks and doesnt get passed into the pdif formula anyway when `cRangedRatio` calls for the `getRangedPDIF` . Hopefully we can rewrite this entire thing in the future to make it more understandable. Definitely a future project.
![image](https://user-images.githubusercontent.com/105882754/204143767-ab01ad23-9145-4adb-8459-794889836c9a.png)

Leaving this note as a reminder in case anyone looks at this when we decide to re-write it.